### DISCUSSION TITLE
Can I raid another stream with my viewers?

### DISCUSSION THREAD

**Discussion by Mrflavor** [KevinBytesTheDust/CVAmp#210] [Q&A]
is there any way i can raid out to any streamer with viewer bots

**Comment by jlplenio** [comment]
Hello you. From what our community has told me, only logged-in users follow raids to other streamers. I am actively working on and maturing the login-functionality - there is a Feature Preview with the login-functionality for supporters over here https://github.com/jlplenio/crude-twitch-viewer-bot/discussions/209.
